docs/Install/Installation.md

```diff
@@ -19,18 +19,37 @@ Requirements
 curl -sS https://getcomposer.org/installer | sudo php -- --install-dir=/usr/bin --filename=composer
 cd /opt/
 sudo git clone https://github.com/librenms/librenmsv2 librenmsv2
-sudo chown -R librenms:librenms /opt/librenmsv2/ -R
 cd librenmsv2
+sudo mkdir logs
+sudo chown -R librenms:librenms /opt/librenmsv2/ -R
+```
+
+```bash
 cp .env.example .env
 ```
 
 Please now edit the .env file and set your options.
 
 ```bash
+replace Travis DB configuration with the one used by Librenms (have a look at the config.php file in /opt/librenms directory)
```

Comment:
Change "replace Travis DB" to "Replace the database" and move this line out of the code block.